Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityController.cs

```diff
@@ -511,8 +525,131 @@ private void UpdateMenuData(InteractionSourceState interactionSourceState, Mixed
             }
         }
 
+        /// <summary>
+        /// Update the Controller Model
+        /// </summary>
+        /// <param name="interactionSourceState"></param>
+        private void UpdateControllerModel(InteractionSourceState interactionSourceState)
+        {
+            if (controllerModelInitialized ||
```

Comment:
>controllerModelInitialized [](start = 16, length = 26)

Once this value is set to true, it looks like UpdateControllerModel will never actually update anyting - is this just maybe misnamed (i.e. it's more like "ensure controller model"). 

Given the current naming I would expect the controller model to update in response to changes, but once this value is true, this entire statement will short circuit and nothing will happen.

Reply:
agreed that this is confusing. will update to EnsureControllerModel